dev-util/codeblocks/codeblocks-20.03-r7.ebuild

```diff
@@ -84,6 +85,14 @@ src_configure() {
 	econf "${myeconfargs[@]}"
 }
 
+src_compile() {
+	if use contrib; then
+		emake -j8  # Bug 930819
```

Comment:
Sorry I didn't say this clearly, but could you implement this the way you originally wanted by utilizing *multiprocessing.eclass*? We shouldn't force `-j8` on people who use say -j4 normally :)

Reply:
I use -j4 in my system globaly and there is no problem with build this particular package after this changes. But I could try to implement via eclass.

Reply:
Done.